core/trino-main/src/main/java/io/trino/operator/Driver.java

```diff
@@ -67,8 +67,6 @@
 
     private final DriverContext driverContext;
     private final List<Operator> activeOperators;
-    // this is present only for debugging
-    @SuppressWarnings("unused")
```

Comment:
It't not obsolete in a sense that nothing changed since then

Reply:
This field is used in `io.trino.operator.Driver#processInternal` method.